integration-test/src/test/java/org/jboss/pnc/integration/WebSocketsNotificationTest.java

```diff
@@ -128,8 +128,8 @@ private void waitForWSClientConnection() {
 
     private void awaitFor(Supplier<Boolean> condition, int timeMs) {
         long waitUntil = System.currentTimeMillis() + timeMs;
-        while (System.currentTimeMillis() < waitUntil) {
-            if (condition.get()) {
+        while(System.currentTimeMillis() < waitUntil) {
```

Comment:
Are you sure you are using the correct formatter? If the answer is yes - that we probably need to sync IntelliJ/Eclipse stuff...